<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mapmaker</class>
 <widget class="QMainWindow" name="Mapmaker">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>587</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>636</width>
    <height>587</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>636</width>
    <height>587</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mapmaker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/resources/maker.png</normaloff>:/resources/maker.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="ViewButton">
          <property name="text">
           <string>View</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/view.png</normaloff>:/resources/view.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AirButton">
          <property name="text">
           <string>Air</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/ice.bmp</normaloff>:/resources/ice.bmp</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BrickButton">
          <property name="text">
           <string>Brick</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/brick.bmp</normaloff>:/resources/brick.bmp</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="IronButton">
          <property name="text">
           <string>Iron</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/iron.bmp</normaloff>:/resources/iron.bmp</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Base1Button">
          <property name="text">
           <string>Base1</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/base1.bmp</normaloff>:/resources/base1.bmp</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Base2Button">
          <property name="text">
           <string>Base2</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/base2.bmp</normaloff>:/resources/base2.bmp</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Spawn1Button">
          <property name="text">
           <string>Spawn1</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/spawn1.bmp</normaloff>:/resources/spawn1.bmp</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Spawn2Button">
          <property name="text">
           <string>Spawn2</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/spawn2.bmp</normaloff>:/resources/spawn2.bmp</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="maptable"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="New"/>
    <addaction name="Open"/>
    <addaction name="Save"/>
    <addaction name="Quit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="Undo"/>
    <addaction name="Redo"/>
   </widget>
   <widget class="QMenu" name="more">
    <property name="title">
     <string>更多</string>
    </property>
    <addaction name="Help"/>
    <addaction name="About"/>
   </widget>
   <addaction name="file"/>
   <addaction name="menu"/>
   <addaction name="more"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="New">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="statusTip">
    <string>新建一个地图文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="statusTip">
    <string>保存当前地图文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="Open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="statusTip">
    <string>打开一个地图文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="Quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="Undo">
   <property name="text">
    <string>撤回</string>
   </property>
   <property name="statusTip">
    <string>撤回一次操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="Redo">
   <property name="text">
    <string>重做</string>
   </property>
   <property name="statusTip">
    <string>重做上次操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="Help">
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
